replica/database.cc

```diff
@@ -921,7 +921,7 @@ future<> database::add_column_family(keyspace& ks, schema_ptr schema, column_fam
     auto&& rs = ks.get_replication_strategy();
     locator::effective_replication_map_ptr erm;
     if (auto pt_rs = rs.maybe_as_per_table()) {
-        erm = pt_rs->make_replication_map(schema->id(), _shared_token_metadata.get());
+        erm = co_await pt_rs->make_replication_map(schema->id(), _shared_token_metadata.get());
```

Comment:
After performing some benchmarks, I decided to remove the preemption point from the `build_dc_replication_factor_map` since with the standard testing parameters of 2048 tablets per table, it takes a half of a microsecond).

Reply:
It turns out my previous benchmark test was incorrect as it was measuring tables with 32 Tablets. I have used `perf-load-balancing` to test the actual time, and for 131072 Tablets the replication factor map building time would be about 9.5ms.  So preemption point should be kept.